index.html

```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```

Comment:
Good job @hasnatali1947 😃  but as the previous reviewer said please ensure that in your README file in the Demo section, you have a link to the deployed project. I currently do not see it in your `application-deploy` branch. Kindly add it in this current branch 👍